webpack.config.js

```diff
@@ -110,8 +110,11 @@ const webpackConfig = {
       {
         'base-uri': "'self'",
         'object-src': "'none'",
-        'script-src': ["'self'"],
+        // NOTE: unsafe-eval is required because of JointJS
+        // https://github.com/sqlectron/sqlectron-gui/issues/663
+        'script-src': ["'self'", "'unsafe-eval'"],
```

Comment:
We talked about not doing this, but I thought about it again and it seemed worse losing the diagram feature just because of the CSP warning. I think it is better just ignore the warning for now, and work on changing the diagram implementation later to avoid this security issue.